code/modules/power/singularity/narsie.dm

```diff
@@ -39,6 +41,29 @@
 /obj/narsie/Initialize(mapload)
 	. = ..()
 
+	narsie_spawn_animation()
+
+/obj/narsie/Destroy()
+	if (GLOB.cult_narsie == src)
+		fall_of_the_harbinger()
+		GLOB.cult_narsie = null
+
+	return ..()
+
+/// This proc sets up all of Nar'Sie's abilities, stats, and begins her round-ending capabilities. She does not do anything unless this proc is invoked.
+/// This is only meant to be invoked after this instance is initialized in specific pro-sumer procs, as it WILL derail the entire round.
+/// Pass in the user that invoked this proc ONLY if it was called via View Variables, if this is invoked through normal gameplay please do pass in null.
+/obj/narsie/proc/start_ending_the_round()
+	var/mob/potential_user = null
+	if(IsAdminAdvancedProcCall())
+		if(tgui_alert(usr, ADMIN_WARNING_MESSAGE, "Begin Nar'Sie Roundender", list("I'm Sure", "Abort")) != "I'm Sure")
```

Comment:
in case you're wondering, no this message doesn't doublesend i did indeed test ✔️ this

it's because the VV_HK invocation of the proc doesn't count as an AdminAdvancedProcCall